README.md

```diff
@@ -1380,11 +1380,12 @@ end
 Note that `enable :sessions` actually stores all data in a cookie. This
 might not always be what you want (storing lots of data will increase your
 traffic, for instance). You can use any Rack session middleware: in order to
-do so, do **not** call `enable :sessions`, but instead pull in your
-middleware of choice as you would any other middleware:
+do so, do **not** call `enable :sessions`, but instead call `set
+:sessions` with your middleware of choice passed in as the value for
+`:session_store` along with any other options:
 
 ```ruby
-use Rack::Session::Pool, :expire_after => 2592000
```

Comment:
If the `use` example still works I'd like to show both (since both should be supported).